cost-analyzer/templates/cost-analyzer-service-template.yaml

```diff
@@ -9,12 +9,12 @@ spec:
     app: cost-analyzer
   type: ClusterIP
   ports:
-    - name: cost-analyzer-server
+    - name: server
```

Comment:
Can these be `kc-server`, `kc-model`, and `kc-frontend` to minimize risk of collisions? +@AjayTripathy

Reply:
I don't figure out where is the risk of collision as all this ids come from the same chart. Can you give more explanations?

Reply:
You can disregard my comment. I was referring to our internal development process but this should be fine.